third_party/move/move-compiler-v2/tests/bytecode-generator/wildcard7.exp

```diff
@@ -30,31 +30,12 @@ public fun m::test(): u8 {
      var $t1: u8
      var $t2: u8
      var $t3: u64
-     var $t4: u8
-     var $t5: u64
   0: $t1 := 40
   1: $t2 := move($t1)
-  2: $t4 := infer($t1)
-  3: $t5 := 30
-  4: $t3 := infer($t5)
-  5: $t0 := infer($t2)
-  6: return $t0
+  2: $t3 := 30
+  3: $t0 := infer($t2)
+  4: return $t0
 }
 
 
-Diagnostics:
-warning: Unused assignment to `q`. Consider removing or prefixing with an underscore: `_q`
-  ┌─ tests/bytecode-generator/wildcard7.move:6:13
-  │
-6 │         let (_, q) = (x, z);
-  │             ^^^^^^
-
-
-Diagnostics:
-error: cannot move local `x` since it is still in use
```

Comment:
Are we okay not showing this error anymore? Is this a change from v1 semantics? If so, perhaps add a transactional test to explicitly show this behavior.